core/creator/src/main/java/io/quarkus/creator/phase/runnerjar/RunnerJarPhase.java

```diff
@@ -346,6 +340,52 @@ public void accept(Path path) {
         }
     }
 
+    /**
+     * Manifest generation is quite simple : we just have to push some attributes in manfest.
+     * However, it gets a little more complex is manifest preexists.
+     * So we first try to see if a manifest exists, and otherwise create a new one.
+     * 
+     * <b>BEWARE</b> this method should be invoked after file copy from target/classes and so on.
+     * Otherwise this manifest manipulation will be useless.
+     * 
+     * @see https://github.com/quarkusio/quarkus/issues/1443
+     * @param runnerZipFs
+     * @param classPath
+     * @throws IOException
```

Comment:
You can remove all the `@` entries, they are useless.

If you squash everything and has the issue id in the commit comment (I usually add "Fixes #xxxx" as a second line), it will serve this purpose with the Git history.

Reply:
I will try the squash (but am not enough git-ready to make sure I will succeed)